src/applications/vaos/project-cheetah/redux/reducer.js

```diff
@@ -477,6 +482,46 @@ export default function projectCheetahReducer(state = initialState, action) {
         },
       };
     }
+    case FORM_PAGE_CONTACT_FACILITIES_OPEN_SUCCEEDED: {
+      let facilities = action.facilities;
+      const address = action.address;
+      const hasResidentialCoordinates =
+        !!action.address?.latitude && !!action.address?.longitude;
+      const sortMethod = hasResidentialCoordinates
+        ? FACILITY_SORT_METHODS.distanceFromResidential
+        : FACILITY_SORT_METHODS.alphabetical;
+
+      if (hasResidentialCoordinates && facilities.length) {
```

Comment:
Even though we can't switch sort methods, the AC has the facilities sorted by residential address, which alphabetical sorting as a fallback.